### PR TITLE
MINOR: Change topic-exists log for CreateTopics from INFO to DEBUG

### PR DESCRIPTION
**Pull request by rajinisivaram**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: core/src/main/scala/kafka/server/AdminManager.scala [hunk 1]
```diff
@@ -179,6 +179,9 @@ class AdminManager(val config: KafkaConfig,
         CreatePartitionsMetadata(topic.name, assignments, ApiError.NONE)
       } catch {
         // Log client errors at a lower level than unexpected exceptions
+        case e: TopicExistsException =>
+          debug(s"Topic '${topic.name}' already exists.", e)
```
Comment (ijuma) [inline core/src/main/scala/kafka/server/AdminManager.scala]: Should we make it clear that this is during topic creation?

Comment (rajinisivaram) [reply core/src/main/scala/kafka/server/AdminManager.scala]: @ijuma Thanks for the review, updated.

**Review by ijuma — APPROVED** [review]
LGTM

**Comment by rajinisivaram** [comment]
@ijuma Thanks for the review, merging to trunk and 2.4

**Pull request merged**